ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/Dockerfiles/Dockerfile.ubi8

```diff
@@ -0,0 +1,30 @@
+FROM redhat/ubi8
+# Install Base Requirements
+RUN dnf -y update && dnf install -y perl openssh-server unzip wget epel-release
+RUN ssh-keygen -t rsa -f /etc/ssh/ssh_host_rsa_key -P ""
+# Install java8 via WGET
+RUN wget -q 'https://api.adoptopenjdk.net/v2/binary/releases/openjdk8?openjdk_impl=hotspot&os=linux&arch=x64&release=latest&type=jdk' -O /tmp/jdk8.tar.gz 
```

Comment:
Using incorrect API, please use api.adoptium.net

Reply:
We should change them together across all of the static dockerfiles since I presume this is what's in the others - I'd probably suggest a separate PR for that but am OK with it being changed here too.

Reply:
I'll change it in this one, then create an issue to update the others.

Reply:
This has been fixed in the latest commit, I've also raised [issue [https://github.com/adoptium/infrastructure/issues/2567](url)](https://github.com/adoptium/infrastructure/issues/2567) for updating the others.